.github/workflows/main.yaml

```diff
@@ -66,13 +64,18 @@ jobs:
           conda-solver: libmamba
           python-version: "${{ matrix.python_version }}"
 
+      # Ensure compatible Poetry version after conda setup
```

Comment:
poetry should be >2 but it is already handled by codna.. so you don't need to add this step